subscription/notify_lambda/logger.py

```diff
@@ -0,0 +1,29 @@
+import os
+import logging
+import sys
+
+LOG_LEVEL = os.getenv("LOG_LEVEL")
+if not LOG_LEVEL:
+    LOG_LEVEL = logging.INFO
+
+def setup_logger(name, log_file=None, level=logging.INFO):
```

Comment:
in general I think more type hints should be used.

Reply:
I have added Type Hints.